### ISSUE TITLE
changefeed stuck, log has "etcdserver: request is too large...owner exited with error"

### ISSUE THREAD

**Issue by Tammyxia** [pingcap/tiflow#2083]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

-  1x changefeed, 3x CDC server, real-time sync.

- sql-gen to write DDL&DML workload to upstream, table count totally is 28464

2. What did you expect to see?

- sync completed.

3. What did you see instead?

- changefeed stuck, cdc log:
![Er8Iyd7wVO](https://user-images.githubusercontent.com/78345569/122387518-869c3300-cfa1-11eb-9a65-453ca8035be7.png)

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
         Release Version: v5.1.0
Edition: Community
Git Commit Hash: 4ca5d03fbe2fe1d2106a29a38b4c99be66b36eb1
Git Branch: heads/refs/tags/v5.1.0
UTC Build Time: 2021-06-17 07:24:39

        ```

    - TiCDC version (execute `cdc version`):

        ```
       [release-version=v5.1.0] [git-hash=a91a10bb5cd52ee22e56217402003a596a1b5f2b] [git-branch=heads/refs/tags/v5.1.0] 
        ```

**Comment by Tammyxia** [comment]
Degrade severity beacause currently capture don't support 20k table count.

**Comment by amyangfei** [comment]
Won't do this in the short term, maybe after October 2021.

**Comment by overvenus** [comment]
We have confirmed this issue is fixed in release-5.4.
